doc/releases/changelog-0.26.0.md

```diff
@@ -108,85 +161,6 @@
-    
-<h4>Classical shadows 👤</h4>
-
-* All-new features for implementing the classical-shadows protocol are now available. 
-  [(#2820)](https://github.com/PennyLaneAI/pennylane/pull/2820)
-  [(#2821)](https://github.com/PennyLaneAI/pennylane/pull/2821)
-  [(#2871)](https://github.com/PennyLaneAI/pennylane/pull/2871)
-  [(#2968)](https://github.com/PennyLaneAI/pennylane/pull/2968)
-  [(#2959)](https://github.com/PennyLaneAI/pennylane/pull/2959)
-  [(#2968)](https://github.com/PennyLaneAI/pennylane/pull/2968)
-
-  The classical-shadow measurement protocol is described in detail in the
-  [classical shadows paper](https://arxiv.org/abs/2002.08953).
-  As part of the support for classical shadows in this release, two new finite-shot and fully-differentiable measurements are available: 
-
-  - QNodes returning `qml.classical_shadow` will return two entities: 
-
-    + `bits`: 0 or 1 if the 1 or -1 eigenvalue is sampled, respectively
-    + `recipes`: the randomized Pauli measurements that are performed for each qubit, identified as a unique integer: 0 for Pauli X, 1 for Pauli Y, and 2 for Pauli Z.
-    
-    ```python
-    dev = qml.device("default.qubit", wires=2, shots=3)
-
-    @qml.qnode(dev)
-    def circuit():
-        qml.Hadamard(wires=0)
-        qml.CNOT(wires=[0, 1])
-        return qml.classical_shadow(wires=[0, 1])
-    ```
-
-    ```pycon
-    >>> bits, recipes = circuit()
-    >>> bits
-    tensor([[0, 0],
-            [1, 0],
-            [0, 1]], dtype=uint8, requires_grad=True)
-    >>> recipes
-    tensor([[2, 2],
-            [0, 2],
-            [0, 2]], dtype=uint8, requires_grad=True)
-    ```
-
-  - QNodes returning `qml.shadow_expval` yield the expectation value estimation using classical shadows:
-
-    ```python
-    dev = qml.device("default.qubit", wires=range(2), shots=10000)
-
-    @qml.qnode(dev)
-    def circuit(x, H):
-        qml.Hadamard(0)
-        qml.CNOT((0,1))
-        qml.RX(x, wires=0)
-        return qml.shadow_expval(H)
-
-    x = np.array(0.5, requires_grad=True) 
-    H = qml.Hamiltonian(
-            [1., 1.], 
-            [qml.PauliZ(0) @ qml.PauliZ(1), qml.PauliX(0) @ qml.PauliX(1)]
-        )  
-    ```
-
-    ```pycon
-    >>> circuit(x, H), 
-    tensor(1.8486, requires_grad=True) 
-    >>> qml.grad(circuit)(x, H))
-    -0.4797000000000001
-    ```
-
-  Fully-differentiable QNode transforms for both new classical-shadows measurements are also available via
-  `qml.shadows.shadow_state` and `qml.shadows.shadow_expval`, respectively.
-  
-  For convenient post-processing, we've also added the ability to calculate general Renyi entropies 
-  by way of the `ClassicalShadow` class' `entropy` method, which requires the wires of the subsystem of interest
-  and the Renyi entropy order:
-
-  ```pycon
-  >>> shadow = ClassicalShadow(bits, recipes)
-  >>> vN_entropy = shadow.entropy(wires=[0, 3], alpha=1)
-  ``` 
 
 <h4>Simplifying just got... simpler 🫰</h4>
```

Comment:
love it 😆 Although the emoji does not render for me: 
![image](https://user-images.githubusercontent.com/2959003/190291394-49e69c6b-7ced-411a-9399-12107844bb8a.png)

Reply:
Some suggestions:

- 'The `qml.simplify` module' ➡ 'The `qml.simplify()` function'
- 'intuitive additions' ➡ do you mean improvements here?
- 'simplify or do' sounds a bit odd to me